src/components/MAPI/clusters/ClusterDetail/ClusterDetailWidgetProvider.tsx

```diff
@@ -84,7 +85,13 @@ const ClusterDetailWidgetProvider: React.FC<IClusterDetailWidgetProviderProps> =
 }) => {
   const region = providerCluster?.spec.location;
 
-  const accountID = providerCluster?.spec.subscriptionID;
+  const accountID = useMemo(() => {
+    if (!providerCluster) return undefined;
+    if (!providerCluster?.spec.subscriptionID) return '';
+
+    return providerCluster?.spec.subscriptionID;
```

Comment:
This is no longer optional due to the falsy value check above

```suggestion
    if (!providerCluster.spec.subscriptionID) return '';

    return providerCluster.spec.subscriptionID;
```